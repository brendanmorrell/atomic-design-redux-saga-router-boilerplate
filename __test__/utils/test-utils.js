import React from 'react';
import { render } from 'react-testing-library';
import { ThemeProvider } from 'styled-components';
import theme from '../../src/theme';
import { Router } from 'react-router';
import { Provider } from 'react-redux';
import { storeFactory, history } from '../../src/store';

export const store = storeFactory({});
const customRender = (node, options) =>
  render(
    <Provider store={store}>
      <Router history={history}>
        <ThemeProvider theme={theme}>{node}</ThemeProvider>
      </Router>
    </Provider>,
    options,
  );

// re-export everything
export * from 'react-testing-library';

// override render method
export { customRender as render };
