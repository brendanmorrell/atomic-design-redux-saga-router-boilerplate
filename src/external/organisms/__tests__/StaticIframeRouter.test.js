import React from 'react';
import { render } from 'test-utils';
import StaticIFrameRouter from '../StaticIframeRouter';
test('snapshot', () => {
  const { container } = render(<StaticIFrameRouter />);
  expect(container.firstChild).toMatchSnapshot();
});
