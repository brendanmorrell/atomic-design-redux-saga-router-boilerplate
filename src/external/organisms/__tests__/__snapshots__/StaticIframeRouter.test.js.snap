// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<iframe
  allowfullscreen=""
  class="IFrame__Iframe-sc-81s4bd-0 jokWXi"
  frameborder="0"
  src="https://www.medmen.com/"
  title="/"
/>
`;
